ui/src/pages/Discord/Servers/NightRunners.tsx

```diff
@@ -7,7 +7,7 @@ export function NightRunners() {
       title="Night Runners"
       description={text}
       img="/images/bunnyL.jpg"
-      joinLink="discord/deeznuts"
+      joinLink="48UXs8NzKF"
```

Comment:
is it possible to put all this information in an object array and then populate it into the template component

Reply:
okay, ill try that

Reply:
I did some thinking i dont think this would be good considering eventually, we would pass in data from description and i dont think that would work well with how we structure our stuff, what do you recommend?

Reply:
I think it would work 
`
{
    "id": 1,
    "title": "Night Runners",
    "description": {text}, // don't this this is correct 
    "img": [],
    "joinLink": "48UXs8NzKF"
}
`

Reply:
ask the wajster @jmorales8

Reply:
remind me tomorrow